doc/xcodebuild.txt

```diff
@@ -1,31 +1,139 @@
-*xcodebuild.txt* Plugin to build and test Xcode projects from vim
+*xcodebuild.txt*
+
+                            xcodebuild.vim
+                Build and test Xcode projects from Vim
 
 ==============================================================================
-CONTENTS                               *xcodebuild-contents*
+CONTENTS                                                         *XCB-Contents*
 
+      1. About.......................... |XCB-About|
```

Comment:
I agree with the general idea of namespacing them to make help lookup easier. In this case I think having the single leading `xcodebuild` is sufficient, rather than repeated.